samples/drivers/sdhc/sdhc_read_write/src/main.c

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) 2025 EXALT Technologies.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <string.h>
+#include <stdio.h>
+#include <assert.h>
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+#include <zephyr/sd/sdio.h>
+#include <zephyr/sys/util.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/drivers/sdhc.h>
+
+LOG_MODULE_REGISTER(sdio_sample, LOG_LEVEL_INF);
+
+static int cmd52_read(const struct device *dev, struct sdhc_command *cmd, enum sdio_func_num func,
+	       uint32_t reg_addr)
+{
+	cmd->opcode = SDIO_RW_DIRECT;
+	cmd->arg = func << SDIO_CMD_ARG_FUNC_NUM_SHIFT |
+		      (reg_addr & SDIO_CMD_ARG_REG_ADDR_MASK) << SDIO_CMD_ARG_REG_ADDR_SHIFT;
```

Comment:
```suggestion
	cmd->arg = func << SDIO_CMD_ARG_FUNC_NUM_SHIFT |
		   ((reg_addr & SDIO_CMD_ARG_REG_ADDR_MASK) << SDIO_CMD_ARG_REG_ADDR_SHIFT);
```
Ditto in `cmd52_write()` definition.